### ISSUE TITLE
JAWS - navigation instruction for menubar

### ISSUE THREAD

**Issue by kolaps33** [FreedomScientific/VFO-standards-support#203]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Go to https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html
> 2. Tab into first item in the menubar example. 

### Expected result
As menu bar is horizontal, user get message like for vertical menu. 

### Actual result
JAWS narrate (take look on the instruction message):
"""
Mythical University menu
About 
1 of 3
To move through items press up or down arrow.
"""

### Example
https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html

#### JAWS version and build number
2019.1904.60
 
#### Operating System and version
Windows 10 Enterprise
Version 1903

#### Browser and version:
Chrome 74.0.3729.169

**Comment by anuvenkatesh1** [comment]
Hi Any updates on this issue? The menubar role is not read by JAWS when first tabbing to the menubar so how is a screen reader user to know this is a menubar with individual drop down menus/submenus?

**Comment by jducrot** [comment]
I've tested confirmed this is still a bug with JAWS 2020.2006.12 in the following browsers:
- Chrome 84
- FireFox 79
- IE 11

In all 3 browsers users are prompted to use "to move through items use up and down arrows". Pressing the down arrow on any of the menubar items triggers the appearance of its menu and focuses the first menuitem. The correct controls to move through the menu bar items are the left and right arrows. If these keys are used, navigation through menubar items happens as expected and JAWS makes the correct announcements.

**Comment by jcorradino** [comment]
Is there an update to this?  This still seems to be a problem.

**Comment by stevefaulkner** [comment]
tested in JAWS 2024.2310.70  The menubar role and aria-label not announced.

works as expected in NVDA

test case: https://cdpn.io/pen/debug/ExrvPwR

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 49596

**Comment by JeremiahRogersVispero** [comment]
Confirmed to work as expected in JAWS 2026.2512.50
